contributing/communications.md

```diff
@@ -38,10 +38,10 @@ We don't have _strict_ guidelines on how you should write. We're not going to at
 
 Follow Plain English guidelines to help safeguard the accessibility of your written content. It'd be ironic for an a11y project to produce documents that are hard to understand! All relevant guidelines for [writing well][writing-well] apply. You might find tools like [Hemingway App][hemingway] helpful when editing. In particular:
 
-- Be concise  
+- Be brief  
```

Comment:
This loses some information, IMO. The two words aren't directly synonymous because "concise" implies a degree of care and efficiency in word choice that "brief" alone doesn't. This one is a bit like trying to replace the word "performant" with "fast" ;)

The only reasonable word I can think of that's a true synonym for "concise" is "succinct", which is helpfully defined by Google as "briefly and clearly expressed". "Brief and clear" might be our saviour here, although it does disrupt my "Be X" subheadings :p